docs/advanced.md

```diff
@@ -538,8 +538,10 @@ var argv = require('yargs')
 ### Using the non-singleton interface
 
 To use yargs without running as a singleton, do:
+
 ```js
 const argv = require('yargs/yargs')(process.argv.slice(2))
 ```
 
-This is especially useful when using yargs in a library, as library authors should not pollute the global state.
+We recommend using this approach by default, as the singleton interface is
```

Comment:
Eventually, all of the docs will use the non-singleton interface `require('yargs/yargs')`. Thus, it might be nice to clarify what the singleton interface is here, to make it clearer which part of the above code block is the singleton/non-singleton interface.

```suggestion
We recommend using this approach by default, as the singleton interface (`require('yargs')`) is
```